### PR TITLE
Discovery Fixes

### PR DESCRIPTION
**Pull request by nisdas**
- [x] Read random nodes to find new nodes from our local table.
- [x] Bump up polling period to 30 seconds to lookup new peers.
- [x] Connect to discovery v5 bootnodes.

### CODE REVIEW

File: beacon-chain/p2p/service.go [hunk 1]
```diff
@@ -30,7 +30,8 @@ import (
 
 var _ = shared.Service(&Service{})
 
-var pollingPeriod = 1 * time.Second
+// check local table every 30 seconds for newly added peers.
+var pollingPeriod = 30 * time.Second
```
Comment (terencechain) [inline beacon-chain/p2p/service.go]: is this formal enough to be defined in beacon's `config.go`?

Comment (nisdas) [reply beacon-chain/p2p/service.go]: Not really, most of these params are only useful in the `p2p` package. This is just a baseline on how often to poll our local table for new peers. I think they are fine living here in this package.

File: beacon-chain/p2p/service.go [hunk 2]
```diff
@@ -26,11 +26,13 @@ import (
 	"github.com/prysmaticlabs/prysm/beacon-chain/p2p/peers"
 	"github.com/prysmaticlabs/prysm/shared"
 	"github.com/prysmaticlabs/prysm/shared/runutil"
+	"github.com/sirupsen/logrus"
 )
 
 var _ = shared.Service(&Service{})
 
-var pollingPeriod = 1 * time.Second
+// check local table every 5 seconds for newly added peers.
```
Comment (terencechain) [inline beacon-chain/p2p/service.go]:
```suggestion
// Check local table every 5 seconds for newly added peers.
```

File: beacon-chain/p2p/service_test.go [hunk 3]
```diff
@@ -130,6 +130,13 @@ func TestListenForNewNodes(t *testing.T) {
 	bootListener := createListener(ipAddr, pkey, cfg)
 	defer bootListener.Close()
 
+	// use shorter period for testing
```
Comment (terencechain) [inline beacon-chain/p2p/service_test.go]:
```suggestion
	// Use shorter period for testing.
```

**Comment by rauljordan** [comment]
Tests fail @nisdas

**Comment by prestonvanloon** [comment]
Discovery unit tests are failing

**Pull request merged**